src/lv_misc/lv_task.h

```diff
@@ -88,23 +88,21 @@ LV_ATTRIBUTE_TASK_HANDLER uint32_t lv_task_handler(void);
 //! @endcond
 
 /**
- * Create an "empty" task. It needs to initialzed with at least
+ * Create an "empty" task. It needs to initialized with at least
  * `lv_task_set_cb` and `lv_task_set_period`
- * @return pointer to the craeted task
+ * @return pointer to the created task
  */
 lv_task_t * lv_task_create_basic(void);
 
 /**
  * Create a new lv_task
- * @param task_xcb a callback which is the task itself. It will be called periodically.
- *                 (the 'x' in the argument name indicates that its not a fully generic function because it not follows
- *                  the `func_name(object, callback, ...)` convention)
+ * @param task_cb a callback which is the task itself. It will be called periodically.
  * @param period call period in ms unit
  * @param prio priority of the task (LV_TASK_PRIO_OFF means the task is stopped)
  * @param user_data custom parameter
  * @return pointer to the new task
  */
-lv_task_t * lv_task_create(lv_task_cb_t task_xcb, uint32_t period, lv_task_prio_t prio, void * user_data);
+lv_task_t * lv_task_create(lv_task_cb_t task_cb, uint32_t period, lv_task_prio_t prio, void * user_data);
```

Comment:
I'm looking at this again (sorry!) and it appears that the header file and C file had different parameter names before. Now I'm not sure which one got used before and was correct. @kisvegabor and @amirgon, can one of you confirm which of these is supposed to be used?

Reply:
Looking here:
```
	/** Generic prototype of "animator" functions.
	 * First parameter is the variable to animate.
	 * Second parameter is the value to set.
	 * Compatible with `lv_xxx_set_yyy(obj, value)` functions
	 * The `x` in `_xcb_t` means its not a fully generic prototype because
	 * it doesn't receive `lv_anim_t *` as its first argument*/
	typedef void (*lv_anim_exec_xcb_t)(void *, lv_anim_value_t);

	/** Same as `lv_anim_exec_xcb_t` but receives `lv_anim_t *` as the first parameter.
	 * It's more consistent but less convenient. Might be used by binding generator functions.*/
	typedef void (*lv_anim_custom_exec_cb_t)(struct _lv_anim_t *, lv_anim_value_t);

	/** Callback to call when the animation is ready*/
	typedef void (*lv_anim_ready_cb_t)(struct _lv_anim_t *);

	/** Callback to call when the animation really stars (considering `delay`)*/
	typedef void (*lv_anim_start_cb_t)(struct _lv_anim_t *);

```
I see that there is one _xcb and the explanation for "x" is that it doesn't contain the corresponding data object in its parameters list...


Then checked how lv_task_create is called:
 
	from _lv_anim_core_init:
	task_cb => void anim_task(lv_task_t * param)

	from lv_async_call:
	task_cb => void lv_async_task_cb(lv_task_t * task)

	from lv_disp_drv_register:
	task_cb => void _lv_disp_refr_task(lv_task_t * task)

	from lv_indev_drv_register
	task_cb => void _lv_indev_read_task(lv_task_t * task)

In all situations it does get as param the task info to be handled, so maybe "task_cb" instead "task_xcb" is ok?

And actually if I look what lv_task_create does with task_(x)cb I see that it just gives it to lv_task_set_cb which has its param called task_cb: no "x" there. So again "task_cb" might be ok... 
Let's wait for SMEs' opinion

Reply:
The comment before `lv_task_create` states this:
```
@param task_xcb a callback which is the task itself. It will be called periodically.
 *                 (the 'x' in the argument name indicates that its not a fully generic function because it not follows
 *                  the `func_name(object, callback, ...)` convention)
```
It's a special case to make binding generation easier. The convention is when a callback is registered there needs to be a `struct` with `user_data` field where some contextual data can be saved by the binding. In ideal case, the first parameter of `lv_task_create()` should be `lv_task_t *` which is not possible because the task is just being created. That's why we introduced `lv_task_create_basic()` and `lv_task_set_cb`.

Reply:
@embeddedt , @DaPa
The `_xcb` is used for callbacks that don't follow the calling conventions [described here](https://github.com/lvgl/lv_binding_micropython#callbacks).

The conventions are not followed in this case, therefore `_xcb` must be retained (or added where missing).
When these conventions are not followed, the function becomes unavailable for Micropython.

Reply:
Thanks, will revert then to xcb asap